### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by starsunny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ Setup your development environment by the following steps.
...
 export REGISTRY=<your docker username>
 ```
+- For AWS ECR
+
+```bash
+export REGISTRY=<accountid>.dkr.ecr.<Region>.amazonaws.com
+```
+- Changes in build/push
+  Replace this "sudo docker login "${REGISTRY}" -u "${DNAME}" -p "${DPASS}";" with
+  aws ecr get-login-password --region <Region> | docker login --username AWS --password-stdin "${REGISTRY}"
```
Comment (mynktl) [inline README.md]: nit: can we format above block with new line and "`" like below?
```suggestion
- Changes in build/push

  Replace this `sudo docker login "${REGISTRY}" -u "${DNAME}" -p "${DPASS}";` with
  `aws ecr get-login-password --region <Region> | docker login --username AWS --password-stdin "${REGISTRY}"`
```

**Review by mynktl — COMMENTED** [review]
Hi @starsunny,

Thanks for the PR! Changes looks good. I've given one minor comment about the formatting. PTAL.

Thanks!

**Comment by starsunny** [comment]
@mynktl It is also failing CI with the same error I raised  in Issues.

**Comment by mynktl** [comment]
> @mynktl It is also failing CI with the same error I raised in Issues.

It seems bitbucket.org/ww/goautoneg is broken.
To resolve this error, you can replace `bitbucket.org/ww/goautoneg` with `github.com/munnerz/goautoneg@latest`, using `go mod edit -replace bitbucket.org/ww/goautoneg@v0.0.0-20120707110453-75cd24fc2f2c=github.com/munnerz/goautoneg@latest`

**Comment by sourabh-kalal** [comment]
@mynktl Please merge.

**Comment by starsunny** [comment]
@mynktl Please merge...

**Review by mynktl — APPROVED** [review]
Changes look good.

Thanks for the PR @starsunny !

**Pull request merged**